src/questions/expression_control.yaml

```diff
@@ -91,4 +91,576 @@
+  content: What is the do-while loop's specialty?
+  hint: noHint
+  explanation: The Do-while loop is executed at least once, but if the condition inside the while or for loop is false, it will not execute.
+  answerKeys:
+    - B
+  subTopics:
+    - "loops-keywords"
+  difficultyLevel: Low
+  choices:
+    - content: It consumes very less gas
+      key: A
+    - content: The loop is executed for atleast one time
+      key: B
+    - content: It can be defined anywhere in the contract
+      key: C
+    - content: The loop will terminate automatically after 1000 iterations
+      key: D
 
+- uuid: 5328ebc5-3dbe-434d-b64d-bc9500796e8d
+  type: SingleChoice
+  content: What keywords are used to define a do-while loop?
+  hint: noHint
+  explanation: The do-while loop is defined using two keywords `do` and `while`.
+  answerKeys:
+    - C
+  subTopics:
+    - "loops-keywords"
+  difficultyLevel: Low
+  choices:
+    - content: For keyword only
+      key: A
+    - content: While keyword only
+      key: B
+    - content: Both do and while keywords
+      key: C
+    - content: Both if and while keywords
+      key: D
+
+- uuid: b4e4a99a-c7c7-4dbf-9ab1-51d2afede138
+  type: SingleChoice
+  content: What will happen if the condition in the do-while loop is false?
+  hint: noHint
+  explanation: The do-while loop is executed for atleast one time. It doesn't matter if the condition is false the loop will execute for one time.
+  answerKeys:
+    - B
+  subTopics:
+    - "loops-keywords"
+  difficultyLevel: Low
+  choices:
+    - content: The loop will not execute at all
+      key: A
+    - content: The loop is executed for one time
+      key: B
+    - content: The program will crash
+      key: C
+    - content: None of the above
+      key: D
+
+- uuid: 97dbf4a5-4f3c-4a06-87c3-110dfc9622a7
+  type: SingleChoice
+  content: What is the purpose of break keyword?
+  hint: Termination
+  explanation: The break keyword is used to stop the execution of a loop or to terminate the loop. It is mostly used with if/else statements.
+  answerKeys:
+    - A
+  subTopics:
+    - "loops-keywords"
+  difficultyLevel: Low
+  choices:
+    - content: To terminate the loop
+      key: A
+    - content: To make a jump for next iteration
+      key: B
+    - content: To stop a function's execution
+      key: C
+    - content: It is used to handle errors
+      key: D
+
+- uuid: 23997bf8-e4f1-4ef4-9fa6-c4b8faec5bfe
+  type: SingleChoice
+  content: What is the use of the continue keyword?
+  hint: noHint
+  explanation: The keyword "continue" is mostly used with conditional statements in order to skip the current iteration of a loop.
+  answerKeys:
+    - D
+  subTopics:
+    - "loops-keywords"
+  difficultyLevel: Low
+  choices:
+    - content: It is used to handle errors
+      key: A
+    - content: Used for making a contract object
+      key: B
+    - content: Used for making external calls
+      key: C
+    - content: Used to skip iteration
```

Comment:
Used to skip rest of the steps in current iteration and start with next iteration